### ISSUE TITLE
Wrong release group on sample files

### ISSUE THREAD

**Issue by duramato** [guessit-io/guessit#516]
`WWIIs.Most.Daring.Raids.S01E04.Storming.Mussolinis.Island.1080p.WEB.h264-EDHD`
For this Scene release's sample, it parses the group wrongfully for the sample file.

As a single file:
```
For: wwiis.most.daring.raids.s01e04.storming.mussolinis.island.1080p.web.h.264-edhd-sample.mkv
GuessIt found: {
    "title": "wwiis most daring raids",
    "season": 1,
    "episode": 4,
    "episode_title": "storming mussolinis island",
    "screen_size": "1080p",
    "format": "WEB-DL",
    "video_codec": "h264",
    "release_group": "edhd-sample",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode"
}
```
Or in the release's directory:
```
For: WWIIs.Most.Daring.Raids.S01E04.Storming.Mussolinis.Island.1080p.WEB.h264-EDHD/wwiis.most.daring.raids.s01e04.storming.mussolinis.island.1080p.web.h.264-edhd-sample.mkv
GuessIt found: {
    "season": 1,
    "episode": 4,
    "episode_title": "Storming Mussolinis Island",
    "screen_size": "1080p",
    "format": "WEB-DL",
    "video_codec": "h264",
    "title": "wwiis most daring raids",
    "release_group": "edhd-sample",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode"
}
```

**Comment by duramato** [comment]
Same thing also happens to proof files,

```
For: dcs.legends.of.tomorrow.s02e01.1080p.bluray.x264-rovers.proof.jpg
GuessIt found: {
    "title": "dcs legends of tomorrow",
    "season": 2,
    "episode": 1,
    "screen_size": "1080p",
    "format": "BluRay",
    "video_codec": "h264",
    "release_group": "rovers.proof.jpg",
    "mimetype": "image/jpeg",
    "type": "episode"
}
```
```
For: dcs.legends.of.tomorrow.s02e01.720p.bluray.x264-demand.sample.mkv
GuessIt found: {
    "title": "dcs legends of tomorrow",
    "season": 2,
    "episode": 1,
    "screen_size": "720p",
    "format": "BluRay",
    "video_codec": "h264",
    "release_group": "demand.sample",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode"
}

```

**Comment by fernandog** [comment]
@Toilal @ratoaq2 are you willing  to add "Sample" in `other`  property ? thanks

**Comment by ratoaq2** [comment]
Fixed by #536
